### PR TITLE
[jecoute]refacto query findDataBySurveyQuestion

### PR DESCRIPTION
**Pull request by Raulnet**

### CODE REVIEW

File: src/Repository/Jecoute/QuestionRepository.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            ->leftJoin('question.choices', 'choices')
+            ->leftJoin('choices.dataAnswers', 'selectedChoices')
+            ->innerJoin('question.surveys', 'surveyQuestion')
+            ->leftJoin('surveyQuestion.dataAnswers', 'dataAnswer')
+            ->andWhere('question = :q')
+            ->setParameter('q', $surveyQuestion->getQuestion())
+            ->groupBy('choices.id')
+            ->getQuery()
+            ->getArrayResult()
+            ;
```
Comment (JeremieSamson) [inline src/Repository/Jecoute/QuestionRepository.php]:
```suggestion
        ;
```

Comment (Raulnet) [reply src/Repository/Jecoute/QuestionRepository.php]: fixed

File: src/Jecoute/StatisticsProvider.php [hunk 2]
```diff
@@ -36,7 +36,7 @@ private function createQuestions(Collection $questions): array
         /** @var SurveyQuestion $surveyQuestion */
         foreach ($questions as $surveyQuestion) {
             $question = $surveyQuestion->getQuestion();
-            $dataBySurveyQuestion = $this->dataAnswerRepository->findDataBySurveyQuestion($surveyQuestion);
+            $dataBySurveyQuestion = $this->questionRepository->findDataBySurveyQuestion($surveyQuestion);
```
Comment (thomasbisignani) [inline src/Jecoute/StatisticsProvider.php]: You can pass directly the `$question` here, and rename `findDataBySurveyQuestion` by `findDataQuestion`

Comment (Raulnet) [reply src/Jecoute/StatisticsProvider.php]: fixed

**Pull request merged**